### PR TITLE
docs: issue #394 Issue Template Updates for Figma link w/signoff

### PR DESCRIPTION
**Pull request by aryanas159**
**Description**

This PR fixes #
issue #394 
**Notes for Reviewers**
Added necessary information as provided in the issue.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [✅ ] Yes, I signed my commits.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -31,6 +31,12 @@ Steps to reproduce the behavior:
 [Optional] **Additional Context**
 Add any other context about the problem here.
 
+- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)```
+
+#### Acceptance Tests
+1. All references to Figma include the "open invite" link.
```
Comment (leecalcote) [inline .github/ISSUE_TEMPLATE/bug_report.md:37]: This section was accidentally included in the issue description. please remove

**Review by leecalcote — CHANGES_REQUESTED** [review]
Thanks!

**Comment by leecalcote** [comment]
@aryanas159 this is missing the desired changes. @yashchaddha will you give @aryanas159 some pointers (show him your PR as an example) and let's get him reassigned to another open one of these issues.

**Comment by yashchaddha** [comment]
Sure @leecalcote i will.

Hi @aryanas159 lets sync up to resolve another similar issue with an effective PR

**Comment by aryanas159** [comment]
@yashchaddha  sure, what should we work on?

**Comment by yashchaddha** [comment]
Hi @aryanas159 lets work on

https://github.com/meshery/meshsync/issues/210

comment there so that @leecalcote  can assign you!

Thanks

**Pull request closed without merging**